python/sglang/srt/model_executor/forward_batch_info.py

```diff
@@ -772,16 +773,24 @@ def prepare_mlp_sync_batch(self, model_runner: ModelRunner):
         # padding
         self._pad_inputs_to_size(model_runner, num_tokens, bs)
         self.global_num_tokens_cpu = global_num_tokens
-        global_num_tokens_pinned = torch.tensor(global_num_tokens, pin_memory=True)
-        self.global_num_tokens_gpu.copy_(global_num_tokens_pinned, non_blocking=True)
+        global_num_tokens_pinned = torch.tensor(
+            global_num_tokens, pin_memory=not _is_cpu
+        )
+        self.global_num_tokens_gpu.copy_(
+            global_num_tokens_pinned, non_blocking=not _is_cpu
+        )
```

Comment:
reduce python if-else as much as possible:
```
self.use_pin_memory = not _is_cpu
```

Reply:
Fixed in https://github.com/sgl-project/sglang/pull/12961/commits/23b3927dab228a6f6d068d647d5ef697ef459816